test/test_podman_pods.sh

```diff
@@ -60,8 +60,8 @@ podman ps --no-trunc | grep $ctrid
 ########
 # Containers in a pod share network namespace
 ########
-podman run -dt --pod foobar quay.io/baude/alpine_nginx:latest
-podman run -it --rm --pod foobar fedora-minimal:28 curl http://localhost
```

Comment:
is there not a muliarch f29 fedora-minimal ?

Reply:
```
docker pull fedora-minimal:29
Error response from daemon: pull access denied for fedora-minimal, repository does not exist or may require 'docker login'
```

Reply:
You need to have the right registries configured.  Remember, any fedora image on docker.io is not a "fedora" image; something we generated.
```
[fedora@lipod-1 ~]$ sudo podman pull fedora-minimal:29
Trying to pull docker.io/fedora-minimal:29...Failed
Trying to pull registry.fedoraproject.org/fedora-minimal:29...Getting image source signatures
Copying blob sha256:5111e6014170938e5360e18c3d820357b86b5b03032a4fca8a6ad3b56c2b3cca
 39.97 MB / 39.97 MB [=====================================================] 17s
Copying config sha256:0d9ba404529baf199cb900d8eaa465c384fe153a1d243ebc79c69ac63d2c229b
 1.27 KB / 1.27 KB [========================================================] 0s
Writing manifest to image destination
Storing signatures
0d9ba404529baf199cb900d8eaa465c384fe153a1d243ebc79c69ac63d2c229b
[fedora@lipod-1 ~]$ sudo podman run --rm -it  fedora-minimal:29 ls
bin  boot  dev	etc  home  lib	lib64  lost+found  media  mnt  opt  proc  root	run  sbin  srv	sys  tmp  usr  var

```

Reply:
now using fq name for fedora `registry.fedoraproject.org/fedora:29`